### ISSUE TITLE
wsj.com: breakage

### ISSUE THREAD

**Issue by xradial** [uBlockOrigin/uAssets#26543]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367), [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) or [_a shortener/hosting site_](https://github.com/uBlockOrigin/uAssets/discussions/17361) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.). Comment in the old issue threads even when they are closed or even if you have a different problem.
- [X] I am not using uBO Lite along with other content blockers.
- [X] I have verified that the issue still exists after using the [`Complete` filtering mode](https://user-images.githubusercontent.com/585534/195342612-85d109d9-9006-4eb5-95a5-fec8a4f233ea.png).
- [X] I have verified that the web browser's built-in content blocker/tracking protection, [network wide/DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking), or my VPN is not causing the issue.
- [X] I have verified that other extensions are not causing the issue.
- [X] If this is about a breakage or detection, I have verified that it is caused by uBOL Lite and isn't a site issue.
- [ ] I did not answer truthfully to **ALL** the above checkboxes.

### URL address of the web page

`https://www.wsj.com/us-news/drones-new-jersey-faa-ban-9b81ed52?`

### Category

breakage

### Description

When the uBOL extension is enabled, and any mode of filtering is selected, the section at the end of WSJ articles where readers make comments is blocked.

### Other extensions used

None

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>

### Configuration

```yaml
version: uBOL 2024.12.15.1269
mode: complete / complete
rulesets:
 default
 block-lan
 dpollock-0
 adguard-spyware-url
 annoyances-cookies
 annoyances-overlays
 annoyances-social
 annoyances-widgets
 annoyances-others
 urlhaus.full
 stevenblack-hosts
```

**Comment by gorhill** [comment]
We can investigate as this would require us to buy a subscription.

See if you can reproduce the issue when enabling _only_ the default ruleset.

**Comment by xradial** [comment]
Hi Raymond,

I’m not sure what you mean by the “default ruleset”.  This behavior occurs whenever the extension is enabled regardless of whether the filtering mode is set at Basic, Optimal, or Complete.  It does not occur when filtering is turned off altogether.  I have a paid subscription to the WSJ, so if you need someone to test your modifications, I’m your guy.

Rich

**Comment by stephenhawk8054** [comment]
You are enabling other filter lists after installing uBOL

```
block-lan
 dpollock-0
 adguard-spyware-url
 annoyances-cookies
 annoyances-overlays
 annoyances-social
 annoyances-widgets
 annoyances-others
 urlhaus.full
 stevenblack-hosts
```

Uninstall and reinstall uBOL. Keep the default settings and test again.

**Comment by xradial** [comment]
That worked!  Thank you.

**Comment by liamengland1** [comment]
Non-sub link `https://www.wsj.com/us-news/drones-new-jersey-faa-ban-9b81ed52?st=b6uQMY&reflink=desktopwebshare_permalink`
